### PR TITLE
Add the missing help output for new command: client capa redirect

### PR DESCRIPTION
**Pull request by hwware**
Update client help output message for new command: client capa redirect

### CODE REVIEW

File: src/networking.c [hunk 1]
```diff
@@ -3485,6 +3485,10 @@ void clientCommand(client *c) {
         const char *help[] = {
 "CACHING (YES|NO)",
 "    Enable/disable tracking of the keys for next command in OPTIN/OPTOUT modes.",
+"CAPA <option> [options...]",
+"    The client claims its some capability options, Options are:",
```
Comment (enjoy-binbin) [inline src/networking.c]:
```suggestion
"    The client claims its some capability options. Options are:",
```

File: src/networking.c [hunk 2]
```diff
@@ -3485,6 +3485,10 @@ void clientCommand(client *c) {
...
+"    * REDIRECT",
+"      The client can handle redirection during primary and replica failover",
```
Comment (enjoy-binbin) [inline src/networking.c]: do we need to mention that the failover is non-cluster mode?

Comment (madolson) [reply src/networking.c]: +1, I think we should clarify that it's for standalone.

Comment (madolson) [inline src/networking.c]:
```suggestion
"      The client can handle redirection during primary and replica failover in standalone mode.",
```

**Review by enjoy-binbin — APPROVED** [review]

**Review by madolson — APPROVED** [review]

**Pull request merged**